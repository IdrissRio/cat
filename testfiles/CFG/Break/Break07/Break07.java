public class Break07 {
  {
  outer:
    do {
      int d;
    inner:
      do {
        int c;
        break outer;
      } while (true);

    } while (true);
    int f;
  }
}