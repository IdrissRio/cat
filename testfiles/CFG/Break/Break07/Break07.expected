NodeName: Entry[15] Successors: [Exit[16]]
NodeName: Exit[16] Successors: []
NodeName: Entry[7] Successors: [VariableDeclarator[40]]
NodeName: VariableDeclarator[40] Successors: [VariableDeclarator[52]]
NodeName: VariableDeclarator[52] Successors: [VariableDeclarator[65]]
NodeName: VariableDeclarator[65] Successors: [Exit[8]]
NodeName: Exit[8] Successors: []

NodeName: Exit[16] Predecessor: [Entry[15]]
NodeName: Entry[15] Predecessor: []
NodeName: Exit[8] Predecessor: [VariableDeclarator[65]]
NodeName: VariableDeclarator[65] Predecessor: [VariableDeclarator[52]]
NodeName: VariableDeclarator[52] Predecessor: [VariableDeclarator[40]]
NodeName: VariableDeclarator[40] Predecessor: [Entry[7]]
NodeName: Entry[7] Predecessor: []
